content/html/concepts/text-formatting/text-formatting.md

```diff
@@ -0,0 +1,43 @@
+
+## HTML Tags
+
+The following HTML tags can affect the formatting of text:
+
+- [`<b>`](https://www.codecademy.com/resources/docs/html/tags/b) Represents text that the reader's attention should be directed to, without signifying any importance. Typically displayed with boldface.
+- [`<i>`](https://www.codecademy.com/resources/docs/html/tags/i) Represents text set off as an idiomatic expression, technical term, taxonomical designation and so on. Typically displayed with italics.
+- [`<u>`](https://www.codecademy.com/resources/docs/html/tags/u) Represents text to be displayed with a non-textual annotation. Typically displayed using a solid underline.
+- [`<strong>`](https://www.codecademy.com/resources/docs/html/tags/strong) Represents text that is urgent or important. Typically displayed with boldface.
+- [`<em>`](https://www.codecademy.com/resources/docs/html/tags/em) Represents text which is emphasized. Typically displayed with italics.
+- [`<h1> - <h6>`](https://www.codecademy.com/resources/docs/html/tags/h1-h6) Represents various heading levels for text. Typically affects font size and weight.
+
+While historically `<b>`,`<i>` and `<u>` tags were used to explicitly format text in a HTML document, that use has been depreciated. These tags now indicate characteristics of the text without specifying a definitive format for how it is rendered. If a specific specified format is required, CSS should be used.
+
+## CSS
+
+To render text with some specific format, CSS is used. CSS can be applied using a stylesheet or using the `style` attribute. (see [CSS Anatomy](https://www.codecademy.com/resources/docs/css/anatomy)) The following CSS properties can be used to format text:
+
+- [`@font-face`](https://www.codecademy.com/resources/docs/css/typography/font-face) Specifies a custom font to be used to display text.
+- [`font`](https://www.codecademy.com/resources/docs/css/typography/font) Shorthand property that sets different properties for an element's font in a single declaration.
+- [`font-family`](https://www.codecademy.com/resources/docs/css/typography/font-family) Specify the typeface in a rule set.
+- [`font-size`](https://www.codecademy.com/resources/docs/css/typography/font-size) Sets the text size. Font size values can be in different units like absolute lengths and relative lengths.
+- [`font-style`](https://www.codecademy.com/resources/docs/css/typography/font-style) To set the font style in which text will appear. Should be used instead of `<i>` to display italics.
+- [`font-weight`](https://www.codecademy.com/resources/docs/css/typography/font-weight) To set the text to be thicker or thinner. Should be used instead of `<b>` to make text bold.
+- [`text-decoration`](https://www.codecademy.com/resources/docs/css/typography/text-decoration) To add lines on the text. Should be used instead of `<u>` to underline text.
+- [`text-shadow`](https://www.codecademy.com/resources/docs/css/typography/text-shadow) Adds shadow to text.
```

Comment:
```suggestion
- [`@font-face`](https://www.codecademy.com/resources/docs/css/typography/font-face): Specifies a custom font to be used to display text.
- [`font`](https://www.codecademy.com/resources/docs/css/typography/font): Shorthand property that sets different properties for an element's font in a single declaration.
- [`font-family`](https://www.codecademy.com/resources/docs/css/typography/font-family): Specify the typeface in a rule set.
- [`font-size`](https://www.codecademy.com/resources/docs/css/typography/font-size): Sets the text size. Font size values can be in different units like absolute lengths and relative lengths.
- [`font-style`](https://www.codecademy.com/resources/docs/css/typography/font-style): To set the font style in which text will appear. Should be used instead of `<i>` to display italics.
- [`font-weight`](https://www.codecademy.com/resources/docs/css/typography/font-weight): To set the text to be thicker or thinner. Should be used instead of `<b>` to make text bold.
- [`text-decoration`](https://www.codecademy.com/resources/docs/css/typography/text-decoration): To add lines on the text. Should be used instead of `<u>` to underline text.
- [`text-shadow`](https://www.codecademy.com/resources/docs/css/typography/text-shadow): Adds shadow to text.
```